src/c++/perf_analyzer/command_line_parser.cc

```diff
@@ -435,7 +436,15 @@ CLParser::Usage(const std::string& msg)
                    "sequence used for sequence models. A sequence with length "
                    "x will be composed of x requests to be sent as the "
                    "elements in the sequence. The length of the actual "
-                   "sequence will be within +/- 20% of the base length.",
+                   "sequence will be within +/- X% of the base length, where X "
+                   "defaults to 20% and is user-settable via "
+                   "`--sequence-length-variation`.",
```

Comment:
Mention that variation only happens when JSON data is not specified.

Reply:
Reworded 👍